### ISSUE TITLE
Generate deep link is locating go.onelink.me

### ISSUE THREAD

**Issue by ViswaDB** [AppsFlyerSDK/appsflyer-react-native-plugin#536]
# Report

## Plugin Version
`6.12.2`

## On what Platform are you having the issue?
 ios and android

## What did you do?
I tried to generate a deep link. Here's the code 
appsFlyer.setAppInviteOneLinkID('8kih');

    appsFlyer.generateInviteLink(
      {
        channel: 'gmail',
        campaign: 'SMS Campaign',
        customerID: '123ref123',
        userParams: {
          referralId: referralId,
        },
      },
      link => {
        console.log('generate link', link);
      },
      err => {
        console.log('generate link error', err);
      },
    );

## What did you expect to happen?
I am expecting the link to be in this format. https://<myappname>.onelink.me

## What happened instead?
But i got https://go.onelink.me.

**Comment by Shiggins95** [comment]
Just commenting here as I had the same issue and contacted support for assistance. Not sure if this will fix the issue for everyone, but it did for me.

You need to add the line for setting the template ID **before** you initialise the SDK. The documentation isn't too clear on this currently, but that was the solution provided by support for me and it worked.

Once the template ID is set, and then you initialise the SDK, you can then call the `generateInviteLink` anywhere in your app and it will generate using the correct subdomain.

**Comment by amit-kremer93** [comment]
@ViswaDB Please try what @Shiggins95 suggested and let us know if it helped

**Comment by MassimoDjungle** [comment]
> @ViswaDB Please try what @Shiggins95 suggested and let us know if it helped

I tried it, but I obtain links to _go.onelink.me_

**Comment by ViswaDB** [comment]
> @ViswaDB Please try what @Shiggins95 suggested and let us know if it helped

Yes after setAppInviteOneLinkID in app js, it's returning correct link in generateInviteLink.

**Comment by akhano26** [comment]
@ViswaDB my link is generated with the correct domain name, but it is not navigating to play store or opening the installed app but my oneLink with the same domain doing both. what could be the reason
